Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
+    }
+    environment {
+        // What variables do we need here
+        projectName = 'DDF'
```

Comment:
In the future I think this is where we would set variables to indicate if this is a pr build, master build or a nightly build.